[2023-06-13T06:45:01.539+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_to_db_dag.add_entry_to_db_task manual__2023-06-13T06:44:59.062908+00:00 [queued]>
[2023-06-13T06:45:01.562+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_to_db_dag.add_entry_to_db_task manual__2023-06-13T06:44:59.062908+00:00 [queued]>
[2023-06-13T06:45:01.564+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-13T06:45:01.564+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-13T06:45:01.566+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-13T06:45:01.600+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): add_entry_to_db_task> on 2023-06-13 06:44:59.062908+00:00
[2023-06-13T06:45:01.616+0000] {standard_task_runner.py:55} INFO - Started process 248 to run task
[2023-06-13T06:45:01.629+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'add_entry_to_db_dag', 'add_entry_to_db_task', 'manual__2023-06-13T06:44:59.062908+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/AssignmentDag.py', '--cfg-path', '/tmp/tmp3t45w11e']
[2023-06-13T06:45:01.633+0000] {standard_task_runner.py:83} INFO - Job 37: Subtask add_entry_to_db_task
[2023-06-13T06:45:01.808+0000] {task_command.py:389} INFO - Running <TaskInstance: add_entry_to_db_dag.add_entry_to_db_task manual__2023-06-13T06:44:59.062908+00:00 [running]> on host 4f6c6e07547c
[2023-06-13T06:45:01.905+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=add_entry_to_db_dag
AIRFLOW_CTX_TASK_ID=add_entry_to_db_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T06:44:59.062908+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-13T06:44:59.062908+00:00
[2023-06-13T06:45:01.915+0000] {base.py:73} INFO - Using connection ID 'postgres_connection' for task execution.
[2023-06-13T06:45:01.927+0000] {sql.py:364} INFO - Running statement: INSERT INTO assignment_table (execution_time) VALUES ('2023-06-13 06:45:01.907614'), parameters: None
[2023-06-13T06:45:01.930+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-13T06:45:01.934+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-13T06:45:01.954+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=add_entry_to_db_dag, task_id=add_entry_to_db_task, execution_date=20230613T064459, start_date=20230613T064501, end_date=20230613T064501
[2023-06-13T06:45:01.998+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-13T06:45:02.018+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
